### ISSUE TITLE
How to run the feature based program

### ISSUE THREAD

**Issue by vinayakumarr** [fernandoandreotti/cinc-challenge2017#3]
There is no main file

**Comment by fernandoandreotti** [comment]
See the [README](https://github.com/fernandoandreotti/cinc-challenge2017/blob/master/featurebased-approach/README.md):

The following steps are necessary to perform the feature-based arrhythmia detection.

1. `ExtractFeatures()` performs feature extraction for each record within a folder

2. `TrainClassifier()` trains an Ensemble of Bagged Decision Trees and Multilayer Perceptron Classifier to classify segments of ECG into the defined categories. Saves resulting classifiers for future usage

3. `PredictTestSet()` makes use of pre-trained classifiers to produce results on test set

**Comment by vinayakumarr** [comment]
It worked
